### PR TITLE
Portfolio Section Finished

### PR DESCRIPTION
**Pull request by Felipeg005**
The next actions were taken:

1. Cellphone adapted, the first section of portfolio template finished
2. Worked with CSS & HTML
3. Edited elements in photoshop for design

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,52 @@
...
+- Browser (Chrome,Firefoxe,IE,Opera,Safari)
+
+### Usage
+
+- Open Html.index file with your browser
+
+
+## Authors
+
+👤 **Author**
```
Comment (arslanbisharat) [inline README.md:30]: - [x] [Optional] You can add author name here

File: index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width"/>
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="header">
+        <div class="headerelement headerelement1">Welcome</div>
+        <div class="headerelement headerelement2"><img src="file:///C:/Users/Juan/Documents/GitHub/Portfolio/Images/nav-menu1.png" height="25" width="25" alt="navimg"/></div>
```
Comment (arslanbisharat) [inline index.html]: - [x] The hamburger/menu button should be enclosed between the `nav` tag. 
- [x] the path should start `./` currently no image is being rendered on the online version of your project.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        <p class="pheadline"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have 
+            a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+    <div class="mediaimgs">
+        <div><img src="file:///C:/Users/Juan/Documents/GitHub/Portfolio/Images/twitter.png" alt="twitter" height="20" width="22"/></div>
+        <div><img src="file:///C:/Users/Juan/Documents/GitHub/Portfolio/Images/linkedin.png" alt="linkedin" height="20" width="22"/></div>
+        <div><img src="file:///C:/Users/Juan/Documents/GitHub/Portfolio/Images/medium.png" alt="medium" height="20" width="22"/></div>
+        <div><img src="file:///C:/Users/Juan/Documents/GitHub/Portfolio/Images/github.png" alt="github" height="20" width="22"/></div>
+        <div><img src="file:///C:/Users/Juan/Documents/GitHub/Portfolio/Images/angellist.png" alt="angellist" height="20" width="22"/></div>
+    </div>
```
Comment (arslanbisharat) [inline index.html]: - [x] All these should be icons and enclosed using `ul` and `li` tags.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  margin: 1px 15px 1px 15px;
+}
+
+.mediaimgs {
+  display: flex;
+  justify-content: space-between;
+  margin: 50px 80px 1px 80px;
+}
+
+#p1 {
```
Comment (arslanbisharat) [inline styles.css]: - [x] Please don't use ID as a selector.  If you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  color: #172b4d;
+}
+
+#p2 {
```
Comment (arslanbisharat) [inline styles.css]: - [x] Please don't use ID as a selector.  If you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi @Felipeg005, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (Felipeg005) [reply README.md:30]: perfect

File: index.html (hunk 2 above)
Comment (Felipeg005) [reply index.html]: Done, thank you

File: index.html (hunk 3 above)
Comment (Felipeg005) [reply index.html]: Great

File: styles.css (hunk 5 above)
Comment (Felipeg005) [reply styles.css]: Ok, it's everything done, thank you, with this, I learned a lot! :smiley:

File: index.html (hunk 2 above)
Comment (rloterh) [reply index.html]: Well reolved

File: index.html (hunk 3 above)
Comment (rloterh) [reply index.html]: Well reolved

File: styles.css (hunk 4 above)
Comment (rloterh) [reply styles.css]: Well resolved

File: styles.css (hunk 5 above)
Comment (rloterh) [reply styles.css]: Well resolved

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Yannick, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 💯 👍.
Congratulations! 🎉 🎉 🥂

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] You can improve the `Author` section of your ReadMe by removing the second author heading,  to make it look like this:
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
__As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).__

File: styles.css (hunk 4 above)
Comment (Felipeg005) [reply styles.css]: Thank you @rloterh

**Pull request merged**